prac_06/guitars.py

```diff
@@ -1,19 +1,75 @@
+"""a program that keeps a list of user guitars and then displayes them once the user enters an empty string"""
 
-VINTAGE_AGE = 50
+from guitar import Guitar
 
 
-class Guitars:
+def main():
+    """get guitars and add them to a list until nothing is entered then display all guitars """
+    guitars = []
+    print("My Guitars!")
+    name, year, cost = get_input()
+    guitar = Guitar(name, cost, year)
+    while guitar.name != "":
+        print("{} added!".format(guitar))
+        guitars.append(guitar)
+        name, year, cost = get_input()
+        guitar = Guitar(name, cost, year)
+    print("\n_______Snip________\n")
+    display_guitars(guitars)
 
-    def __init__(self, name="", cost=0, year=0):
-        self.cost = cost
-        self.name = name
-        self.year = year
 
-    def __str__(self):
-        return "{} ({}) : {:.2}".format(self.name, self.year, self.cost)
+def display_guitars(guitars):
+    """display all guitars"""
+    for i, guitar in enumerate(guitars, 1):
+        vintage_string = ""
+        if guitar.is_vintage():
```

Comment:
Could consider using
vintage_string = '(vintage)' if guitar.is_vintage() else ''
rather than the if statement